### PR TITLE
Updated to .csproj and ver 2.02

### PR DESCRIPTION
**Pull request by exiled78**

### CODE REVIEW

**Comment by hikalkan** [comment]
Thank you for your contribution.

**Comment by exiled78** [comment]
@hikalkan - No worries. Sorry, but I did not mean to leave this file in the PR : https://github.com/aspnetboilerplate/aspnetboilerplate-samples/blob/master/SimpleTaskSystem-Core/src/Acme.SimpleTaskApp.Core/Tasks/TaskWriter.cs

You may wish to remove as it was just for some testing. Also, I have found in my own projects that I have needed to add assembly bindings like : 
https://github.com/aspnetboilerplate/aspnetboilerplate-samples/blob/master/SimpleTaskSystem-Core/test/Acme.SimpleTaskApp.Tests/app.config

I see that you removed them from the web project. I wonder what I am doing wrong that I get "..expected version 1.1.1 but found 1.1.2 type errors. Any tips? Thanks.

**Comment by hikalkan** [comment]
Hi,

In the web project, we have ``<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>``: https://github.com/aspnetboilerplate/aspnetboilerplate-samples/blob/master/SimpleTaskSystem-Core/src/Acme.SimpleTaskApp.Web/Acme.SimpleTaskApp.Web.csproj#L10

We can add it to the test project too.

I will remove TaskWriter.cs, thanks.

File: SimpleTaskSystem-Core/src/Acme.SimpleTaskApp.Core/Tasks/TaskWriter.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            _mgr = mgr;
+            _repo = repo;
+        }
+
+        public void HandleEvent(
+            Abp.Events.Bus.Entities.EntityCreatingEventData<Acme.SimpleTaskApp.Tasks.Task> eventData)
+        {
+            Debug.WriteLine("*** Task Created ***");
+            var result = _repo.GetAll();
```
Comment (exiled78) [inline SimpleTaskSystem-Core/src/Acme.SimpleTaskApp.Core/Tasks/TaskWriter.cs]: @hikalkan - as per my message

**Comment by exiled78** [comment]
@hikalkan if you don't mind, I will ask you another question. That class (TaskWriter.cs) was being used to replicate an issue I am facing with my project. I found that in the handler, when trying to use the repository, it would error as being disposed. Not sure if you can advise the reason or a better approach? 

This disposed state can be found at TaskWriter.cs line 34 : https://github.com/aspnetboilerplate/aspnetboilerplate-samples/pull/27/commits/9120cccf8dee6a6292c4b1b188d3258386d9abaf

I added a comment to this line for your reference.

Thanks!

**Comment by hikalkan** [comment]
Thank you for reporting. Can you create another issue for that?

**Comment by exiled78** [comment]
@hikalkan - Please see https://github.com/aspnetboilerplate/aspnetboilerplate-samples/issues/28
Thanks!

**Pull request merged**